lib/ObjWriter/objwriter.exports

```diff
@@ -1,6 +1,7 @@
 InitObjWriter
 FinishObjWriter
 SwitchSection
+SwitchSectionEx
```

Comment:
Yeah I can do that. When I wrote this last week I rationalized that we call SwitchSection often so shouldn't eat the extra interop overhead. But marshalling 0 and null is cheap so I'll just collapse them into one API.